src/char/int_mail.c

```diff
@@ -337,7 +337,11 @@ static bool inter_mail_return_message(int char_id, int mail_id, int *new_mail)
 		safestrncpy(msg.dest_name, temp_, NAME_LENGTH);
 
 		// set reply message title
-		snprintf_ignore_truncation(temp_, MAIL_TITLE_LENGTH, "RE:%s", msg.title); // String truncation expected and ignored
+		PRAGMA_GCC7(GCC diagnostic push)
```

Comment:
i not sure, but may be pragma can works inside macro?

Reply:
they could, with the alternate _Pragma syntax - but I didn't like the macro at all, it made it too tempting to use it instead of resolving an actual issue with the code

Reply:
ok